metomi/isodatetime/tests/test_main.py

```diff
@@ -24,8 +24,8 @@
 from unittest.mock import patch
 
 
-import isodatetime
-import isodatetime.main
+import metomi.isodatetime as isodatetime
```

Comment:
A alias that does not clash with the ``isodatetime`` submodule name would be safer here, as there is the potential for future confusion or errors.

But, since other than ``isodatetime_main`` there is only one call to it, there seems little need to define an alias, just import what is required:
```suggestion
from metomi.isodatetime import __version__
```
then use that method directly on line 42.

Reply:
Yes.